packages/account/package.json

```diff
@@ -39,6 +39,7 @@
         "@deriv/hooks": "^1.0.0",
         "@deriv/integration": "1.0.0",
         "@deriv/quill-icons": "^1.22.12",
+        "@deriv-com/quill-ui": "^1.13.10",
```

Comment:
@yauheni-deriv  Please kindly remove the caret ^ from quill-ui version in all package.jsons where it's installed🙏🏻
Please refer to a comment from another PR where are removing it to fully control the version updates from now on: https://github.com/binary-com/deriv-app/pull/15828#discussion_r1670271736

Reply:
thanks @maryia-deriv for letting me know this 🙇